app/utilities/messages.py

```diff
@@ -21,3 +21,15 @@
     status.HTTP_200_OK: {"description": "User recovered"},
     status.HTTP_404_NOT_FOUND: {"description": "User not found"},
 }
+
+GET_PUBLIC_KEY_RESPONSES = {
+    status.HTTP_201_CREATED: {"description": "User Public key saved"}
+}
```

Comment:
Si se usa en un POST, no debería ser:
```suggestion
POST_PUBLIC_KEY_RESPONSES = {
    status.HTTP_201_CREATED: {"description": "User Public key saved"}
}
```
?
O tiene que ver con lo que comentaste en el PR?

Reply:
Se me olvido cambiarlo ahi lo soluciono